makefile-docker.defs

```diff
@@ -72,12 +72,52 @@ build:
 test_bashscripts: build_bats
 	docker run --rm -v "${PWD}/.github:/code" bats-with-helpers:latest /code/tests/test_retag_and_push.bats
 
+vespa_confirm_cli_installed:
+	@if [ ! $$(which vespa) ]; then \
+		echo 'ERROR: The vespa cli is not installed, please install and try again:' ; \
+		echo 'https://docs.vespa.ai/en/vespa-cli.html'; \
+		exit 1; \
+	fi
+
+vespa_healthy:
+	@if [ ! $$(curl -f -s 'http://localhost:19071/status.html') ]; then \
+		echo 'ERROR: Bad response from local vespa cluster, is it running?'; \
+		exit 1; \
+	fi
+
+VESPA_TEST_APP_DIR='tests/search_fixtures/vespa_test_schema'
```

Comment:
I'm not sure if you're use of variables in a makefile is correct and is actually doing what is intended. Just a flag to make sure we are using the right type of the two flavours of vars: https://www.gnu.org/software/make/manual/html_node/Flavors.html

Reply:
At one point this was being used by more than one target, but it actually doesnt need to be a variable anymore, so I can sidestep this 😁

Reply:
Done